### ISSUE TITLE
Page Language is Not Declared Accurately During Internationalization

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#5013]
The default language of Open Library is declared on the main `html` element. While the displayed language can be changed the the options in the footer (beneath `id="footer-locale-menu"`), this does not programmatically change the language declared in the HTML element. This content consistently declares the page language as English (via `lang="en"`).

### Evidence / Screenshot
![Two open library screenshots side by side. The left is the page in English, the right is the page in Spanish. Both programmatically declare the page language as English](https://user-images.githubusercontent.com/3421881/114077898-23817800-985d-11eb-8e18-11a031ae1523.png)

### Relevant url?
Any Open Library page!

### Steps to Reproduce
1. Load the page in English, verify the `html` element has `lang="en"` applied to it
2. Navigate to the bottom of the page & select a different language
3. Verify though the displayed language of the page has changed, the programmatic language has not

* Actual: `lang="en"` is consistently present even when this value should be different
* Expected: The `lang` attribute must be updated to a relevant value (e.g. `es` for Spanish, `cs` for Czech, `de` for German, etc)
    * All of the relevant values are listed next to the languages in the footer menu!

### Details

- **Logged in:** either state
- **Browser type/version:** Chrome & FF Tested (likely all)
- **Operating system:** Windows Tested (likely all)
- **Environment:** prod

### Proposal & Constraints
We should update the `lang` attributes to the relevant value when we update the internationalization of the site

### Related files
- openlibrary/plugins/openlibrary/js/ol.js
- openlibrary/templates/lib/nav_foot.html

### Stakeholders
@bpmcneilly

**Comment by Yashs911** [comment]
@bpmcneilly I would like to work on this issue.

**Comment by Yashs911** [comment]
@mekarpeles I guess we can close this issue.
